source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -314,17 +316,62 @@ all data.
 
 ### Engine database changes
 
-XXX Add disk attachment incremental setting.
+Add backup column to disk_attachments table. Use to mark an image for
+incremental backup.
+
+- disk_attachments
+  - backup: (incremental|null)
+
+Add vm_backups table. This table keep the information about running
+backups tasks. Use during backup to track and montior backup.
+
+- vm_backups
+  - checkpoint_id: UUID
```

Comment:
maybe it would be simpler to have also an 'id' here, and 'backup_id' in 'disk_backup_map'